bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/link/LinkHandler.java

```diff
@@ -149,6 +157,36 @@ private String validateAndResolveLinkTarget(String linkTarget) {
         }
     }
 
+    /**
+     * Validates and resolves the link accessibility label.
+     * @param linkAccessibilityLabel Link accessibility label
+     *
+     * @return The validated link accessibility label or {@code null} if not valid
+     */
+    private String validateAndResolveLinkAccessibilityLabel(String linkAccessibilityLabel) {
```

Comment:
As this method only validates the label, I'd call it `validateLinkAccessibilityLabel`